domain/src/main/java/uk/gov/moj/sdt/domain/IndividualRequest.java

```diff
@@ -121,18 +123,23 @@ public class IndividualRequest extends AbstractDomainObject implements IIndividu
      * Target Application Response for Individual Request processing.
      */
     @Column(name = "TARGET_APPLICATION_RESPONSE")
+    @Lob
+    @Type(type = "org.hibernate.type.BinaryType")
     private byte[] targetApplicationResponse;
 
     /**
      * Error log.
      */
-    @OneToOne(cascade = CascadeType.ALL, targetEntity = ErrorLog.class, mappedBy = "individualRequest")
+    @OneToOne(cascade = CascadeType.ALL, targetEntity= ErrorLog.class)
+    @JoinColumn(name="ERROR_LOG_ID")
```

Comment:
I think the old version (the one with the mappedBy attribute and the JoinColumn annotation removed) is the one that's needed here.

Reply:
I think it is needed to populate the errorLog object